### PR TITLE
Fixed país word

### PR DESCRIPTION
**Pull request by LuisReinoso**
I remove some words that end in letter 's' because they was plural nouns, and finally add 'pais' word to this list.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
-.idea
\ No newline at end of file
+.idea
+package-lock.json
```
Comment (morhook) [inline .gitignore:3]: You shouldn't ignore the `package-lock.json`, as it helps reproduce the state of this npm library.

**Comment by morhook** [comment]
I think #4 is fixing this problem better (as it uses the spanish rule and `país` is not an exception). Can we close this PR and merge the other one?

**Comment by LuisReinoso** [comment]
Sure, it's better, thanks @morhook

**Comment by morhook** [comment]
Can you close this PR @LuisReinoso  ?

**Comment by LuisReinoso** [comment]
Sure :)

**Pull request closed without merging**